developer-community/tech-talks.md

```diff
@@ -0,0 +1,240 @@
+# The full story
+
+# Speakers
+## Contact
+
+Ideally, we should start contacting speakers __2.5 months__ prior to the event.
+
+Our tech talks involve a total of __5 long talks__ and __2 lightning__. Someone from our team should always give either one Lightning or a Long talk. Hence, we should make an initial list of _6 people to contact_. If no reply, _follow-up after 3-5 days_ until you get an answer. 
```

Comment:
4 emails is fair I believe..
1. Initial
2. Did you have time to check?
3. Just wanted to update you on the fact that speaker X accepted to come.
4. Feel free to reach out at a later point if you change your mind.

Thoughts?